.github/workflows/mcp-coderabbit.yml

```
@@ -14,26 +19,37 @@ jobs:
+          OUTPUT=$(node mcp-server.js)
+          echo "$OUTPUT" > mcp-output.json
+
+          # Escape output for GitHub
+          esc_output="${OUTPUT//'%'/'%25'}"
+          esc_output="${esc_output//$'\n'/'%0A'}"
+          esc_output="${esc_output//$'\r'/'%0D'}"
+          echo "result=$esc_output" >> $GITHUB_OUTPUT
+
+      - name: Comment on Pull Request
+        if: ${{ github.event_name == 'pull_request' }}
         uses: actions/github-script@v6
```

actionlint (1.7.9)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)